tests/cupy_tests/linalg_tests/test_decomposition.py

```diff
@@ -203,3 +203,16 @@ def test_svd_no_uv(self):
     def test_rank2(self):
         self.check_rank2(cupy.random.randn(2, 3, 4).astype(numpy.float32))
         self.check_rank2(cupy.random.randn(1, 2, 3, 4).astype(numpy.float64))
+
+    @testing.with_requires('numpy>=1.16')
+    def test_empty_array(self):
+        self.check_usv((0, 3))
+        self.check_usv((3, 0))
+        self.check_usv((1, 0))
+
+    @testing.with_requires('numpy>=1.16')
+    @testing.numpy_cupy_allclose(atol=1e-4)
```

Comment:
In this test, the result arrays can be compared without tolerances.
```suggestion
    @testing.numpy_cupy_array_equal()
```

Reply:
I have made that change.